vector/src/main/AndroidManifest.xml

```diff
@@ -180,7 +180,9 @@
         <activity android:name=".features.createdirect.CreateDirectRoomActivity" />
         <activity android:name=".features.invite.InviteUsersToRoomActivity" />
         <activity android:name=".features.webview.VectorWebViewActivity" />
-        <activity android:name=".features.link.LinkHandlerActivity">
+        <activity
+            android:name=".features.link.LinkHandlerActivity"
```

Comment:
Can you add explicit `enable="true"` attribute? It will be nice for the forks who want to disable it.
Also maybe add a comment to explain which links type are handled here.